plugins/gatsby-source-garden-content/utils.ts

```diff
@@ -113,10 +114,20 @@ export async function fetchFigmaImages({
   nodeIds: string[];
   scale: number;
 }) {
-  const idsParam = nodeIds.join(',');
-  const { images } = await fetchFigmaAPI(`images/${fileId}?ids=${idsParam}&scale=${scale}`, {
-    figmaApiToken
-  });
+  const chunks = chunk(nodeIds, 40);
+  let images: Record<string, string> = {};
+
+  for await (const value of chunks) {
```

Comment:
Maybe a little comment to explain why we have to follow this approach?

Reply:
Apologies, I'll add a description shortly. I meant to keep this in draft until I got more answers from Figma's tech support. 😅